plugins.yml

```diff
@@ -1,41 +1,6 @@
 # This file contains the list of officially endorsed plugins.
```

Comment:
Doesn't seem like we can update this file without breaking Yarn v3
https://github.com/yarnpkg/berry/blob/b433198b9d2fb6ab81a9e23e8f00021bc5bafb4d/packages/plugin-essentials/sources/commands/plugin/list.ts#L6